modules/update-upgrading-sro.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-clusters-sro.adoc
+
+:_content-type: PROCEDURE
+[id="updating-clusters-sro_{context}"]
+= Running a preflight check for the SRO
+
+The following is an example procedure for checking the compatibility of a kernel module version before upgrading a cluster.
+
+.Prerequisites
+
+* You have a running {product-title} cluster.
+* You installed the OpenShift CLI (oc).
+* You are logged into the OpenShift CLI as a user with cluster-admin privileges.
+* You installed the SRO.
+
+.Procedure
+
+. Create the following CRD and save the YAML as `prevalidation.yaml`.
++
+[source,yaml]
+----
+apiVersion: sro.openshift.io/v1beta1
+kind: PreflightValidation
+metadata:
+ name: preflight
+ namespace: preflight
+spec:
+ updateImage: quay.io/openshift-release-dev/ocp-release@sha256:f7f252c39b64601c8ac3de737a584ba4f6016b1f4b17801d726ca2fd15492878 <1>
+----
+<1> Specify the name of the update image here.
++
+. Start the validation check by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f prevalidation.yaml
+----
+
+.Verification
+. Check the status of the CR by running the following command:
++
+[source,terminal]
+----
+$ oc describe preflightvalidations.sro.openshift.io/v1beta1 preflight
+----
++
+.Example output
++
+[source,terminal]
+----
+Status:
+  Cr Statuses:
+    Last Transition Time:  2022-08-02T08:48:45Z
+    Name:                  simple-oot
```

Comment:
Giving that we don't specify anything related to `simple-oot`, maybe we can say that this is a specialresource that was previously deployed on the cluster, or _'Example for preflight validation when `simple-oot` specialresource is deployed_' 

WDYT ?

Reply:
> The following is an example output in which "simple-oot" is a `SpecialResource` CR that has been deployed on the cluster.

Reply:
/lgtm